Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +199,19 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        tab["HEAD"] = formula_last_commit_hash
+        tab["time"] = formula_last_commit_time.to_i
+        tab.write
```

Comment:
> The reason is we may introduce new core code which will generate different content. Therefore, core code version matters in reproducible bottles.

Also: this is fine for reproducibility as we can debug this relatively easily or store this information in e.g. the bottle commit/DSL. The problem with including it in the tap is that the bottle SHA256 changes unnecessarily every time the core HEAD changes.